test/conformance/testing/include/uur/fixtures.h

```diff
@@ -192,6 +192,47 @@ struct urQueueTest : urContextTest {
     ur_queue_handle_t queue = nullptr;
 };
 
+struct urHostPipeTest : urQueueTest {
+    void SetUp() override {
+        UUR_RETURN_ON_FATAL_FAILURE(urQueueTest::SetUp());
+        uur::KernelsEnvironment::instance->LoadSource("foo", 0, il_binary);
+        ASSERT_SUCCESS(urProgramCreateWithIL(
+            context, il_binary->data(), il_binary->size(), nullptr, &program));
+
+        size_t size = 0;
+        ASSERT_SUCCESS(urDeviceGetInfo(
+            device, UR_DEVICE_INFO_HOST_PIPE_RW_SUPPORTED, 0, nullptr, &size));
```

Comment:
I think if `UR_DEVICE_INFO_HOST_PIPE_RW_SUPPORTED` is not supported we could skip the test with:
```
if (!supported) {
        GTEST_SKIP() << "[Feature tested] is not supported.";
}
```